src/mastra/workflows/logCoreAnalyzer/constants.ts

```diff
@@ -1,14 +1,13 @@
 /**
- * Constants for the Log Core Analyzer workflow
+ * Input source types
  */
+export enum SOURCE_TYPE {
+  FILE = 'file',
+  URL = 'url',
+  TEXT = 'text',
+}
```

Comment:
[nit] Use [standard enum casing](https://www.typescriptlang.org/docs/handbook/enums.html)

```suggestion
export enum SourceType {
  File = 'file',
  URL = 'url',
  Text = 'text',
}
```